src/stactools/hotosm/cli.py

```diff
@@ -92,15 +123,139 @@ def create_pgstac_from_ctx(
     callback=create_pgstac_from_ctx,
 )
 
+dump_to_path = click.option(
+    "--file",
+    type=click.Path(
+        writable=True,
+        path_type=Path,
+    ),
+    help="Dump STAC Items to this file.",
+    required=True,
+)
+
+
+catalog_option = click.option(
+    "--catalog",
+    type=click.Choice(["OAM", "Maxar"]),
+    required=True,
+    help="Sync STAC Collection definition for this dataset catalog.",
+)
+
 
+# ===== CLI commands
 @click.group
 def main():
     """STAC for Humanitarian OpenStreetMap Team OpenAerialMap."""
 
 
 @main.command()
+@dump_to_path
+@catalog_option
+def dump_collection(
+    file: Path,
+    catalog: str,
+    **_pgstac_options: Any,
+) -> None:
+    """Dump Collection definition to JSON.
+
+    The output of this CLI program can be used with the `pypgstac load collections` CLI
+    command to bulk load STAC Items into PgSTAC.
+    """
+    create_and_save_collection(catalog, file)
+    click.echo(f"Saved the STAC Collection definition for {catalog} to {file}.")
+
+
+@main.command()
+@catalog_option
+@pgstac_username
+@pgstac_password
+@pgstac_host
+@pgstac_port
+@pgstac_database
+@click.pass_context
+def sync_collection(
```

Comment:
(also discussed in the PR description under "Alternatives Considered")

The "sync to PgSTAC" and "dump to JSON" CLI programs could've been combined, but they have a lot of mutually exclusive options. Rather than adding complexity to the user interface, it seemed nicer and overall less complicated to keep the CLIs separate. To avoid repeating code this added a little bit of complexity in making helper functions, but I think this is a better tradeoff